dealership/vehicles.py

```diff
@@ -1,15 +1,43 @@
 class Vehicle(object):
     def __init__(self, maker, model, year, base_price, miles):
-        pass
+        self.maker = maker
+        self.model = model
+        self.year = year
+        self.base_price = base_price
+        self.miles = miles
 
+    def sale_price(self):
+        try:
+            self.sale_multiplier
+        except AttributeError:
+            print('Missing sale multiplier for vehicle type.')
+        else:
```

Comment:
Don't catch these types of exceptions. The only thing you're doing is just silencing real problems in your application. **Let things crash**. Having errors is a good thing, we can fix those bugs as soon as we see them. And even if you want to catch the exception, `print` is not a good idea. As a general rule of thumb: good, production-ready code has NO prints at all.